src/AppInstallerCommonCore/Manifest/ManifestYamlPopulator.cpp

```diff
@@ -821,6 +876,19 @@ namespace AppInstaller::Manifest
                 installer.AppsAndFeaturesEntries = manifest.DefaultInstallerInfo.AppsAndFeaturesEntries;
             }
 
+            if (installer.InstallerType == InstallerTypeEnum::Zip)

```

Comment:
Use semantic `IsArchiveType` function here as well.

Reply:
Changed to using `IsArchiveType` function